[2025-10-13T08:42:02.360+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T08:42:02.384+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers manual__2025-10-13T08:41:42.376320+00:00 [queued]>
[2025-10-13T08:42:02.392+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers manual__2025-10-13T08:41:42.376320+00:00 [queued]>
[2025-10-13T08:42:02.393+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T08:42:02.408+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_customers> on 2025-10-13 08:41:42.376320+00:00
[2025-10-13T08:42:02.417+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=395) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T08:42:02.419+0000] {standard_task_runner.py:72} INFO - Started process 400 to run task
[2025-10-13T08:42:02.423+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_customers', 'manual__2025-10-13T08:41:42.376320+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpdmm_w37w']
[2025-10-13T08:42:02.425+0000] {standard_task_runner.py:105} INFO - Job 60: Subtask extract_load_group.load_customers
[2025-10-13T08:42:02.481+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_customers manual__2025-10-13T08:41:42.376320+00:00 [running]> on host 84d5a327cf64
[2025-10-13T08:42:02.580+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-10-13T08:41:42.376320+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-13T08:41:42.376320+00:00'
[2025-10-13T08:42:02.582+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T08:42:13.529+0000] {load.py:30} INFO - Loaded 0 rows and 5 columns to gcpl-469311.datalabs_techtest.stg_customers
[2025-10-13T08:42:13.530+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T08:42:13.543+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T08:42:13.544+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_customers, run_id=manual__2025-10-13T08:41:42.376320+00:00, execution_date=20251013T084142, start_date=20251013T084202, end_date=20251013T084213
[2025-10-13T08:42:13.605+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T08:42:13.628+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T08:42:13.630+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
